libraries/ms-helper/inc/generic_can.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+// Module for abstracting CAN implementation details.
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "generic_can_msg.h"
+#include "status.h"
+
+#define GENERIC_CAN_EMPTY_MASK UINT32_MAX
+#define NUM_GENERIC_CAN_RX_HANDLERS 5
+
+struct GenericCan;
+
+typedef void (*GenericCanRx)(const GenericCanMsg *msg, void *context);
+
+typedef struct GenericCanRxStorage {
+  uint32_t filter;
+  uint32_t mask;
+  GenericCanRx rx_handler;
+  void *context;
+} GenericCanRxStorage;
+
+typedef struct GenericCanInterface {
+  StatusCode (*tx)(const struct GenericCan *can, const GenericCanMsg *msg);
+  // Doesn't support ACKable messages (defaults to enabled).
```

Comment:
When you say defaults to enabled, does that mean "returns ok"?

Reply:
Yes. Clarified comment.